config/schemas/config.yaml

```
@@ -14,6 +14,220 @@ $defs:
     thing:
       $ref: "#/$defs/thing"
     ```
+  affinity:
+    $comment: Real k8s resources might have existing schemas that could be reused as-is
+    additionalProperties:
+      type: object
+    default:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: node-role.kubernetes.io/control-plane
+                  operator: Exists
+    title: Affinity
```

```suggestion
    title: Kubernetes Affinity
```
Same reasoning as above. Also can we have a standard sorting within schema so it is easier to read?

Suggested:
- title
- description
- type
- additionalProperties - (since this seems to be the way to extend from other defs)
- properties
- items
- required
- default
- examples

---

Yeah, the order when keys are sorted alphabetically is unfortunate. Do you happen to know we can do this automatically with some variant of `yq4 'sort_keys(..)` ?